servicebroker_node/config/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+	"port":8080, 
+	
+	"service_id": "autoscaler-guid",
```

Comment:
it used to configure the service information and plan information which may different per deployment. i.e. using different service name, service guid and etc.